client.go

```diff
@@ -386,6 +391,10 @@ type Options struct {
 	TracerProvider               trace.TracerProvider
 	MeterProvider                metric.MeterProvider
 
+	// SSH authentication
+	SSHKeyFile       string // path to SSH private key file
```

Comment:
Lets use
```go
import (
   cryptossh "golang.org/x/crypto/ssh"
)

// ...

// SSH authentication.
SSHSigner cryptossh.Signer
```
Here to allow manually loading SSH keys from vaults/memory.

Reply:
done